File: go/1/main.go
// main.go
package main

func main() {
	carro1 := Carro{marca: "Toyota", modelo: "Corolla", ano: 2020}

	carro1.ExibirDetalhes()
	
	carro1.Acelerar(30)
	carro1.ExibirVelocidade()

	carro1.Acelerar(20)
	carro1.ExibirVelocidade()

	carro1.Frear(10)
	carro1.ExibirVelocidade()

	carro1.Frear(50)
	carro1.ExibirVelocidade()
}

File: python/1/main.py
from Carro import *

if __name__ == "__main__":
    carro1 = Carro("Toyota", "Corolla", 2020)

    carro1.exibir_detalhes()

    carro1.acelerar(30)
    carro1.exibir_velocidade()

    carro1.acelerar(20)
    carro1.exibir_velocidade()

    carro1.frear(10)
    carro1.exibir_velocidade()

    carro1.frear(50)
    carro1.exibir_velocidade()
